kommunicate/src/main/java/io/kommunicate/users/KmAssigneeListHelper.java

```diff
@@ -0,0 +1,71 @@
+package io.kommunicate.users;
+
+import android.content.Context;
+
+import com.applozic.mobicomkit.feed.ErrorResponseFeed;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.kommunicate.Kommunicate;
+import io.kommunicate.callbacks.KMGetContactsHandler;
+
+public class KmAssigneeListHelper {
+    private static List<KmContact> agentList;
```

Comment:
can this be changed to:
Map<Integer, <List<KmContact>
where key - type of the user record

then within the same object, we will be able to store and reuse the code.

.getUsers(Type = agent)
.getUsers(Type = bot)

this will remove lot of duplicate code added for fetching and setting this

Reply:
ok

Reply:
done